app/database/schema/server/test.ts

```diff
@@ -42,8 +44,45 @@ describe('*** Test schema for SERVER database ***', () => {
         expect(serverSchema).toEqual({
             version: 1,
             tables: {
+                [CATEGORY]: {
+                    name: CATEGORY,
+                    unsafeSql: undefined,
+                    columns: {
+                        display_name: {name: 'display_name', type: 'string'},
+                        type: {name: 'type', type: 'string'},
+                        sort_order: {name: 'sort_order', type: 'number'},
+                        sorting: {name: 'sorting', type: 'string'},
+                        muted: {name: 'muted', type: 'boolean'},
+                        collapsed: {name: 'collapsed', type: 'boolean'},
+                        team_id: {name: 'team_id', type: 'string', isIndexed: true},
+                    },
+                    columnArray: [
+                        {name: 'display_name', type: 'string'},
+                        {name: 'type', type: 'string'},
+                        {name: 'sort_order', type: 'number'},
+                        {name: 'sorting', type: 'string'},
+                        {name: 'muted', type: 'boolean'},
+                        {name: 'collapsed', type: 'boolean'},
+                        {name: 'team_id', type: 'string', isIndexed: true},
+                    ],
+                },
+                [CATEGORY_CHANNEL]: {
+                    name: CATEGORY_CHANNEL,
+                    unsafeSql: undefined,
+                    columns: {
+                        category_id: {name: 'category_id', type: 'string', isIndexed: true},
+                        channel_id: {name: 'channel_id', type: 'string', isIndexed: true},
+                        sort_order: {name: 'sort_order', type: 'number'},
+                    },
+                    columnArray: [
+                        {name: 'category_id', type: 'string', isIndexed: true},
+                        {name: 'channel_id', type: 'string', isIndexed: true},
+                        {name: 'sort_order', type: 'number'},
+                    ],
+                },
                 [CHANNEL_INFO]: {
                     name: CHANNEL_INFO,
+                    unsafeSql: undefined,
```

Comment:
why is this all of the sudden needed, here and in every other table

Reply:
I'm not sure why but `npm run test` failed for me with not having this in.